src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){
+        return false;
```

Comment:
We shouldn't assign a default behavior to the parent class even though its generally true/false
E.g. what if we assign `canFly()` to be true by default and we forgot to override it for some animals such as penguin?
result -> suddenly penguin can fly